scholarx/archive/functions.js

```diff
@@ -2,6 +2,18 @@ $(function () {
     loadNavAndFooter('/assets/content/static');  //relative path to content directory
 });
 
+let mentor = {
+    2019:[],
+    2020:[],
+    2021:[]
+}
+let mentee = {
+    2019:[],
+    2020:[],
+    2021:[]
+}
+let mentorProfilesData;
+let menteeProfilesData;
```

Comment:
Can you add a new line here @janithlahirukariyawasam 
```suggestion
let menteeProfilesData;

```